gnome/gstreamer1-gst-plugins-base/Portfile

```
@@ -63,8 +63,9 @@ configure.args-append \
                     -Dxshm=disabled \
                     -Dxvideo=disabled
 
-# gstbasetextoverlay.c:1511: error: 'for' loop initial declaration used outside C99 mode
-configure.cflags-append -std=c99
+# as of orc 0.4.34 compilation with c11 standard is required to prevent slugish behavior
+# see https://gitlab.freedesktop.org/gstreamer/orc/-/issues/47
+configure.cflags-append -std=c11
 
 # gst-libs/gst/gl/gstgldebug.h:28: error: redefinition of typedef ‘GstGLAsyncDebug’
 compiler.blacklist-append *gcc-3.* *gcc-4.* {clang < 212}
```

Set `compiler.c_standard 2011` to ensure MacPorts picks a compiler that supports that standard. You can remove the `compiler.blacklist` since the all of the compilers named there do not support C11 and won't be chosen anymore.
```suggestion
compiler.c_standard 2011
configure.cflags-append -std=c11
```

---

Thank you! I made the change accordingly.